2021-10-13 00:09:21,274 INFO    MainThread:23024 [wandb_setup.py:_flush():69] setting env: {}
2021-10-13 00:09:21,274 INFO    MainThread:23024 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-13 00:09:21,274 INFO    MainThread:23024 [wandb_init.py:_log_setup():342] Logging user logs to f:\Lab\nfs\base-visob\wandb\run-20211013_000921-29y7gzzo\logs\debug.log
2021-10-13 00:09:21,274 INFO    MainThread:23024 [wandb_init.py:_log_setup():343] Logging internal logs to f:\Lab\nfs\base-visob\wandb\run-20211013_000921-29y7gzzo\logs\debug-internal.log
2021-10-13 00:09:21,275 INFO    MainThread:23024 [wandb_init.py:init():375] calling init triggers
2021-10-13 00:09:21,275 INFO    MainThread:23024 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-13 00:09:21,275 INFO    MainThread:23024 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 1vsodjtq
2021-10-13 00:09:21,276 INFO    MainThread:23024 [wandb_run.py:finish():1271] finishing run sreeraj/UFPR-Ocular/1vsodjtq
2021-10-13 00:09:21,277 INFO    MainThread:23024 [jupyter.py:save_history():353] not saving jupyter history
2021-10-13 00:09:21,277 INFO    MainThread:23024 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-13 00:09:21,277 INFO    MainThread:23024 [wandb_init.py:_jupyter_teardown():275] cleaning up jupyter logic
2021-10-13 00:09:21,277 INFO    MainThread:23024 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-10-13 00:09:21,277 INFO    MainThread:23024 [wandb_run.py:_restore():1543] restore
2021-10-13 00:09:22,149 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 10438
  total_bytes: 10438
}

2021-10-13 00:09:22,272 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 10438
  total_bytes: 10438
}

2021-10-13 00:09:22,425 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 10438
  total_bytes: 10438
}

2021-10-13 00:09:23,147 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 10438
  total_bytes: 16186
}

2021-10-13 00:09:23,263 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 10438
  total_bytes: 16186
}

2021-10-13 00:09:23,369 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 15893
  total_bytes: 16186
}

2021-10-13 00:09:23,477 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 16186
  total_bytes: 16186
}

2021-10-13 00:09:23,587 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 16186
  total_bytes: 16186
}

2021-10-13 00:09:23,695 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 16186
  total_bytes: 16186
}

2021-10-13 00:09:23,804 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 16186
  total_bytes: 16186
}

2021-10-13 00:09:23,915 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 16186
  total_bytes: 16186
}

2021-10-13 00:09:24,116 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 16186
  total_bytes: 16186
}

2021-10-13 00:09:24,224 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 16186
  total_bytes: 16186
}

2021-10-13 00:09:25,599 INFO    MainThread:23024 [wandb_run.py:_show_summary():1848] rendering summary
2021-10-13 00:09:25,600 INFO    MainThread:23024 [wandb_run.py:_show_history():1886] rendering history
2021-10-13 00:09:25,602 INFO    MainThread:23024 [wandb_run.py:_show_files():1915] logging synced files
2021-10-13 00:09:25,610 INFO    MainThread:23024 [wandb_init.py:init():424] starting backend
2021-10-13 00:09:25,611 INFO    MainThread:23024 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-10-13 00:09:25,614 INFO    MainThread:23024 [backend.py:ensure_launched():135] starting backend process...
2021-10-13 00:09:25,816 INFO    MainThread:23024 [backend.py:ensure_launched():139] started backend process with pid: 32532
2021-10-13 00:09:25,816 INFO    MainThread:23024 [wandb_init.py:init():429] backend started and connected
2021-10-13 00:09:25,818 INFO    MainThread:23024 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-10-13 00:09:25,818 INFO    MainThread:23024 [wandb_run.py:_label_probe_notebook():822] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-10-13 00:09:25,818 INFO    MainThread:23024 [wandb_init.py:init():477] updated telemetry
2021-10-13 00:09:25,818 INFO    MainThread:23024 [wandb_init.py:init():500] communicating current version
2021-10-13 00:09:28,453 INFO    MainThread:23024 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-13 00:09:28,455 INFO    MainThread:23024 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-13 00:09:28,639 INFO    MainThread:23024 [wandb_init.py:init():540] starting run threads in backend
2021-10-13 00:09:32,198 INFO    MainThread:23024 [wandb_run.py:_console_start():1601] atexit reg
2021-10-13 00:09:32,201 INFO    MainThread:23024 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-10-13 00:09:32,201 INFO    MainThread:23024 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-10-13 00:09:32,203 INFO    MainThread:23024 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-13 00:09:32,203 INFO    MainThread:23024 [wandb_init.py:init():565] run started, returning control to user process
2021-10-13 00:23:14,308 INFO    MainThread:23024 [wandb_setup.py:_flush():69] setting env: {}
2021-10-13 00:23:14,308 INFO    MainThread:23024 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-13 00:23:14,308 INFO    MainThread:23024 [wandb_init.py:_log_setup():342] Logging user logs to f:\Lab\nfs\base-visob\wandb\run-20211013_002314-368pqd13\logs\debug.log
2021-10-13 00:23:14,309 INFO    MainThread:23024 [wandb_init.py:_log_setup():343] Logging internal logs to f:\Lab\nfs\base-visob\wandb\run-20211013_002314-368pqd13\logs\debug-internal.log
2021-10-13 00:23:14,309 INFO    MainThread:23024 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001F02B5DCD00>
2021-10-13 00:23:14,310 INFO    MainThread:23024 [wandb_init.py:init():375] calling init triggers
2021-10-13 00:23:14,310 INFO    MainThread:23024 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-13 00:23:14,310 INFO    MainThread:23024 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 29y7gzzo
2021-10-13 00:23:14,312 INFO    MainThread:23024 [wandb_run.py:finish():1271] finishing run sreeraj/UFPR-Ocular/29y7gzzo
